homeassistant/components/twinkly/light.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         # They are expected to be updated using the device_info.
         self._name = conf.data[CONF_NAME] or "Twinkly light"
         self._model = conf.data[CONF_MODEL]
+        self._mac = device_info["mac"]
```

Comment:
I only have one device so I'm not sure if `mac` is present in the payload for all models. If not, please let me know and I'll change the code.

Reply:
Seems to be there for my device (I have a whole bunch of them) 👍

Reply:
Thanks for checking